Added dissenter extension to default apps
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -1348,7 +1348,7 @@ group("extra_resources") {
   }
 }
 
-if (is_chrome_branded && !is_android) {
+if (true) { # is_chrome_branded && !is_android) {
   if (!is_mac) {
     _default_apps_target_type = "copy"
   } else {
@@ -1365,11 +1365,12 @@ if (is_chrome_branded && !is_android) {
     }
 
     sources = [
-      "browser/resources/default_apps/docs.crx",
-      "browser/resources/default_apps/drive.crx",
+      # "browser/resources/default_apps/docs.crx",
+      # "browser/resources/default_apps/drive.crx",
       "browser/resources/default_apps/external_extensions.json",
-      "browser/resources/default_apps/gmail.crx",
-      "browser/resources/default_apps/youtube.crx",
+      # "browser/resources/default_apps/gmail.crx",
+      # "browser/resources/default_apps/youtube.crx",
+      "browser/resources/default_apps/dissenter.crx",
     ]
 
     if (!is_mac) {
--- a/chrome/browser/resources/default_apps/external_extensions.json
+++ b/chrome/browser/resources/default_apps/external_extensions.json
@@ -38,6 +38,10 @@
   // Drive extension
   "ghbmnnjooekpmoecnnnilnnbdlolhkhi" : {
     "external_update_url": "trk:04:https://clients2.google.com/service/update2/crx"
+  },
+  "mhomeaadffipapncjcgmmcldklcoiigj" : {
+    "external_crx": "dissenter.crx",
+    "external_update_url": "trk:03:https://clients2.google.com/service/update2/crx"
   }
 }
 
